<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCLViewer</class>
 <widget class="QMainWindow" name="PCLViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1143</width>
    <height>781</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PCLViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKWidget" name="qvtkWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>600</width>
      <height>340</height>
     </rect>
    </property>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>600</width>
      <height>340</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="camWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>455</width>
      <height>339</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>180</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Video Feed</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>370</y>
      <width>455</width>
      <height>340</height>
     </rect>
    </property>
   </widget>
   <zorder>label_7</zorder>
   <zorder>qvtkWidget</zorder>
   <zorder>qvtkWidget_2</zorder>
   <zorder>camWidget</zorder>
   <zorder>textEdit</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1143</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="actionOpenCapturedFile"/>
     <addaction name="actionLiveStream"/>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionRecord"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionCsv"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenCapturedFile"/>
   <addaction name="actionLiveStream"/>
   <addaction name="actionStop"/>
   <addaction name="actionPreferences"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionPlay"/>
   <addaction name="actionRecord"/>
   <addaction name="actionCsv"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/delete_red.png</normaloff>:/images/delete_red.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpenCapturedFile">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/WiresharkDoc.png</normaloff>:/images/WiresharkDoc.png</iconset>
   </property>
   <property name="text">
    <string>Captured File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionLiveStream">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Lidar.jpg</normaloff>:/images/Lidar.jpg</iconset>
   </property>
   <property name="text">
    <string>Sensor Stream</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/settings2.png</normaloff>:/images/settings2.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/stopSymbol.png</normaloff>:/images/stopSymbol.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop simulation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/media-playback-pause.png</normaloff>
     <normalon>../images/media-playback-pause.png</normalon>:/images/media-playback-pause.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Play/Pause</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/media-record.png</normaloff>
     <normalon>../images/media-record.png</normalon>:/images/media-record.png</iconset>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCsv">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/csv.png</normaloff>:/images/csv.png</iconset>
   </property>
   <property name="text">
    <string>csv</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
